client/galaxy/style/less/ui.less

```diff
@@ -152,7 +152,7 @@
             font-size: 0.8em;
             font-weight: normal;
             top: -1px;
-            word-break:break-word;
```

Comment:
It did but not in FF. Its a webkit feature: http://caniuse.com/#feat=word-break. Thanks for looking into it.